LanguageExt.Core/Try.cs

```diff
@@ -706,20 +706,20 @@ public static Try<T> Filter<T>(this Try<T> self, Func<T, bool> Succ, Func<Except
                 var resU = bind(resT.Value).Try();
                 if (resU.IsFaulted)
                 {
-                    if (resT.Value is ILinqDisposable) (resT.Value as ILinqDisposable).Dispose();
+                    //if (resT.Value is ILinqDisposable) (resT.Value as ILinqDisposable).Dispose();
```

Comment:
We shouldn't check in commented-out code... if it's dead, that's what GIT is for...